webClient/src/app/editor/project-tree/project-tree.component.ts

```diff
@@ -130,24 +156,24 @@ export class ProjectTreeComponent implements OnInit {
     this.editorControl.openDirectory.subscribe(dirName => {
       //Note: This temporary hack is used to hide datasets using the original slower Editor structure.
       // Will be removed when Dataset functionality for Explorer gets better.
-        this.fileExplorer.showUss();
-        this.fileExplorer.updateDirectory(dirName);
-        this.showDatasets = false;
+      this.fileExplorer.showUss();
+      this.fileExplorer.updateDirectory(dirName);
+      this.showDatasets = false;
     });
 
     this.editorControl.openDataset.subscribe(dirName => {
       if (dirName != null && dirName !== '') {
         if (dirName[0] == '/') {
           //Note: This temporary hack is used to hide datasets using the original slower Editor structure.
           // Will be removed when Dataset functionality for Explorer gets better.
-            this.fileExplorer.showUss();
-            this.fileExplorer.updateDirectory(dirName);
-            this.showDatasets = false;
+          this.fileExplorer.showUss();
```

Comment:
Generally, we try to shy away from whitespace changes as to not clog up Git any more than it has to be. Personally, I don't have a problem with it but just for future things.

Reply:
can you share your style guides - like prettier if you are using one, my eslint vscode are formating it automatically, it tried to fix most of them before commit, but then if i modify something again, and hit save, it auto formats, which i like to use

Reply:
This obviously needs to be expanded: https://zowe.org/code-guidelines-2/
But, generally speaking we just want to minimize the amount of changed lines to maintain git history (the "blame" function).
We tend to use 2 spaces per tab, but I know some other groups may use 4, and it's not as important which you use so much as that you maintain consistency with whatever the current file already uses, and that you don't accidentally cause whitespace changes.